helpers/View.php

```diff
@@ -125,12 +129,28 @@ public function jsonSuccess($data) {
     }
 
     /**
+     * returns max mtime for file paths given in array
+     *
+     * @param filePaths array of file paths
+     * @returns max mtime as int (unix timestamp)
+     */
+    public static function maxmtime($filePaths) {
```

Comment:
`array` type hint should be added since it is supported since PHP 5.1: https://3v4l.org/U4Gq4